setup.go

```diff
@@ -68,7 +68,11 @@ func SetupLogging() {
 	zapCfg.OutputPaths = []string{"stderr"}
 	// check if we log to a file
 	if logfp := os.Getenv(envLoggingFile); len(logfp) > 0 {
-		zapCfg.OutputPaths = append(zapCfg.OutputPaths, logfp)
+		if path, err := normalizePath(logfp); err != nil {
+			fmt.Fprintf(os.Stderr, "error normalizing log path '%q': %s\n", logfp, err)
```

Comment:
should output paths revert to just stderr in this case? that doesn't seem obvious based on the printed error.

Reply:
Hm. Yeah, it probably should.

Reply:
Actually, that's what happens. I guess I can just add that to the message.